next.config.mjs

```diff
@@ -7,6 +10,14 @@ const nextConfig = {
   eslint: {
     ignoreDuringBuilds: true,
   },
+  images: {
+    remotePatterns: hostnames.map((hostname) => ({
+      protocol: 'https',
+      hostname,
+      port: '',
+      pathname: '/**',
+    })),
+  },
```

Comment:
do we use this for anything else other than the OP image? if not, please find a [svg of the OP image](https://github.com/ethereum-optimism/brand-kit) and use it instead.